src/client/components/input-login/input-login.style.css

```diff
@@ -0,0 +1,43 @@
+.login {
+  position: relative;
+}
+
+.login input {
+  padding-left: 4em;
+}
+
+.login i {
+  background-color: var(--grey);
+  border-radius: 2.2em 0 0 2.2em;
+  color: #666;
+  left: 0px;
+  position: absolute;
+  padding: 1.1em 1.1em 1em 1.2em;
+  transition: 0.3s;
+  top: 1.2rem;
+}
+
+.login i svg {
+  margin: 0 !important;
+}
+
+.login input:focus + i,
+.login input:focus + i path {
+  background: var(--fontawesome-gray);
+  color: var(--white);
+}
+
+.login input.error {
+  background-color: transparent;
+  box-shadow: 0 0 0 2px inset var(--danger-color);
+}
+
+.login input.error + i {
+  background: var(--danger-color);
+  color: var(--white);
+  padding-bottom: 1em;
+}
+
+.login input.error + i path {
+  color: var(--white) !important;
+}
```

Comment:
Looks quite hacky with the `!important`. Why is this necessary? Are you styling an svg?